manticore/core/executor.py

```diff
@@ -491,6 +491,15 @@ def generate_testcase(self, state, message = 'Testcase generated'):
         
         file(self._getFilename('test_%08x.syscalls'%test_number),'a').write(repr(state.platform.syscall_trace))
 
+        # save symbolic files
+        if hasattr(state.platform, 'files'):
```

Comment:
Consider using files = getattr(state.platform, 'files',None) instead of hasattr. (https://hynek.me/articles/hasattr/)